### PR TITLE
Extensions: WPSC: Accept files object from stats endpoint instead of array

### PR DESCRIPTION
**Pull request by ockham**
To be used with https://github.com/Automattic/wp-super-cache/commit/907ec9a909d3de27963ed9de4df0a2dd398d7086.

Purports to simplify https://github.com/Automattic/wp-calypso/pull/13818/files#diff-597825fe096a64181e9fcfb31718b3fbR96.

To test -- verify that the 'Stale WP-Cached Files'/'Stale Super Cached Files' Cards in the 'Contents' tab still work.

### CODE REVIEW

File: client/extensions/wp-super-cache/cache-stats.jsx [hunk 1]
```diff
@@ -55,18 +56,18 @@ class CacheStats extends Component {
...
-							<td className="wp-super-cache__stat-age">{ getAge( file ) }</td>
-							<td className="wp-super-cache__stat-action">
-								<Button compact>
-									{ translate( 'Delete' ) }
-								</Button>
-							</td>
-						</tr>
-					) }
+						{ map( files, ( file, dir ) => (
+							<tr className="wp-super-cache__stat" key={ file.lower_age }>
```
Comment (donnapep) [inline client/extensions/wp-super-cache/cache-stats.jsx]: It seems conceivable that there may be more than one file with the same `lower_age`, which would of course make the keys non-unique.

Comment (ockham) [inline client/extensions/wp-super-cache/cache-stats.jsx]: D'oh, this is a leftover from some weird debugging that I did 😳  Good spot, this should be `file.url`.

Comment (ockham) [inline client/extensions/wp-super-cache/cache-stats.jsx]: Fixed by 7a974c1.

**Comment by donnapep** [comment]
LGTM 👍

**Pull request merged**